Packs/XMCO/Integrations/Serenety/Serenety.yml

```diff
@@ -0,0 +1,61 @@
+fromversion: 6.10.0
+category: Vulnerability Management
+commonfields:
+  id: Serenety
+  version: -1
+configuration:
+- display: XMCO LePortail instance URL
+  defaultvalue: https://leportail.xmco.fr/
+  name: url
+  required: true
+  type: 0
+- displaypassword: API Key
+  name: api_key
+  required: true
+  type: 9
+  hiddenusername: true
+  section: Connect
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+- display: Maximum number of incidents per fetch
+  defaultvalue: '50'
+  name: max_fetch
+  type: 0
+  required: false
+- defaultvalue: 3 days
+  display: First fetch time
+  additionalinfo: First alert created date to fetch.
+  name: first_fetch
+  type: 0
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+description: 'Fetch Serenety alert from XMCO.'
+display: Serenety
+name: Serenety
+script:
+  commands:
+  - name: fetch-incidents
+    description: Fetch Serenety alert from XMCO LePortail.
+  dockerimage: demisto/python3:3.10.14.92207
+  feed: false
+  isfetch: true
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+tests:
+- No tests (auto formatted).
+
```

Comment:
You can set your mapper as the default for this integration. More info [here](https://xsoar.pan.dev/docs/incidents/incident-classification-mapping#:~:text=You%20can%20set%20default%20classifier%20and/or%20mapper%20for%20an%20integration%20by%20populating%20the%20following%20keys%20in%20the%20integration%20YAML%20file%20with%20the%20classifier%20and/or%20mapper%20IDs%3A).